backend/src/modules/Boards/repository/inMemory/inMemoryBoards.repository.spec.js

```diff
@@ -63,32 +76,97 @@ describe('InMemoryRepository |', () => {
     const foundBoards = testRepository.findBoardByName(testBoard1.boardName);
 
     //THEN
-    console.log('TEST');
-    console.log(foundBoards);
     expect(foundBoards).toContain(testBoard1);
   });
-  test(`throw an error when found board by id don't exist`, () => {
+
+  test('properly add announcement id to board', async () => {
     //GIVEN
     const testRepository = new InMemoryBoardsRepository();
 
     //WHEN
-    const tryFindNotExistingBoard = () => {
-      testRepository.findBoardByID(testBoard1.id);
+    testRepository.createNewBoard(testBoard1);
```

Comment:
to bym przeniósł do given, bo jak testujemy dodawanie id announcementów do tablicy to zakładamy, że tablica jest już gotowa